js/data.js

```diff
@@ -51,41 +51,30 @@ function sparql2GeoJSON(input) {
 }
 
 
+
 function reservesStyle(feature) {
    return {
        fillColor: feature.properties.no > feature.properties.yes ? 'red' : 'green',
        weight: 2,
        opacity: 1,
        color: 'white',
-       fillOpacity: 0.7
+       fillOpacity: getOpacity(feature.properties.totalVoters)
    };
 }
 
-function highlightFeature(feature){
-    var layer = feature.target;
-    layer.setStyle({
-        weight : 2,
-        color : 'black',
-        fillOpacity : 0.6
-    }
-    );
-    layer.bringToFront();
-}
+function getOpacity(voters) {
```

Comment:
See comment above. If we already have a fraction, we can just set the opacity as that fraction.